config/dj_config.py

```diff
@@ -3,21 +3,33 @@
 import os
 import pathlib
 import sys
+import yaml
+import tempfile
 
 import datajoint as dj
 
+def generate_config_yaml(filename: str, **kwargs):
```

Comment:
Is there are reason for having these as kwargs vs. specifying the keyword arguments (and their defaults) directly?